### ISSUE TITLE
is MeteorListView's reactive?

### ISSUE THREAD

**Issue by rafaelcorreiapoli** [inProgress-team/react-native-meteor#41]
Hi. I'm trying to do a simple app just for testing purposes

I have a publication on my meteor server:

```
Meteor.publish('pokemons', function(type) {
  let query = type ? {
    type: type
  } : {};

  return Pokemons.find(query);
});
```

On my RN app:
```
let types = [
  'fire',
  'water',
  'grass'
]
```

```
  constructor(props) {
    super(props);
    this.state = {
      currentType: 0
    }
  }
```

```
  startMeteorSubscriptions() {
    Meteor.subscribe('pokemons', types[this.state.currentType]);
  }
```

```
  _handlePress() {
    let currentType = this.state.currentType;
    currentType++;
    currentType = currentType % types.length;

    this.setState({
      currentType
    })
  }
```

The subscription is changing correctly when the state is changed... but:
```
        <MeteorListView
          collection="pokemons"
          selector={{type: types[this.state.currentType]}}
          renderRow={this.renderItem}
        />
```

The selector on MeteorListView does not respond to changes on the state... it keeps filtering by the first selector passed to it (types[0] -> 'fire')

**Comment by Mokto** [comment]
Oh you are absolutely right !

Thank you for warning me.

Fixing it soon ;)

**Comment by Mokto** [comment]
Done ! Available in 1.0.0-beta21

**Comment by rafaelcorreiapoli** [comment]
Thank you! Glad to know that I contributed to this awesome project
